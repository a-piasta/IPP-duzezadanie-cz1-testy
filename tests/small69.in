getRouteDescription;537
addRoad;wcmcdsawqvsabzwm;yjlnkddrpeibszzsaudxyndqe;15;4
repairRoad;vlvcigpqvnstt;iajlzgkitf;-6
repairRoad;wcmcdsawqvsabzwm;vlvcigpqvnstt;15
repairRoad;iajlzgkitf;inwzyddfqonasnfkbpmmqya;0
repairRoad;inwzyddfqonasnfkbpmmqya;dytyvtighulmna;11
addRoad;inwzyddfqonasnfkbpmmqya;bxxxiltdgdbjwmuddbxvpvkjaky;17;-8
removeRoad;bxxxiltdgdbjwmuddbxvpvkjaky;inwzyddfqonasnfkbpmmqya
extendRoute;50;ajplfdfljikeafqlhclbkllxlrhvl
repairRoad;wbkxadzwix;wcmcdsawqvsabzwm;7
newRoute;275;inwzyddfqonasnfkbpmmqya;vlvcigpqvnstt
getRouteDescription;275
removeRoad;inwzyddfqonasnfkbpmmqya;bxxxiltdgdbjwmuddbxvpvkjaky
addRoad;bxxxiltdgdbjwmuddbxvpvkjaky;vlvcigpqvnstt;13;1
newRoute;158;vlvcigpqvnstt;iajlzgkitf
addRoad;bxxxiltdgdbjwmuddbxvpvkjaky;bxxxiltdgdbjwmuddbxvpvkjaky;5;-14
addRoad;inwzyddfqonasnfkbpmmqya;wbkxadzwix;23;-3
getRouteDescription;158
newRoute;512;iajlzgkitf;bxxxiltdgdbjwmuddbxvpvkjaky
addRoad;vlvcigpqvnstt;vlvcigpqvnstt;16;-6
removeRoad;yjlnkddrpeibszzsaudxyndqe;wcmcdsawqvsabzwm
extendRoute;512;vlvcigpqvnstt
extendRoute;158;ovvtiycrjjhrfrpowpufamrwa
getRouteDescription;158
removeRoad;inwzyddfqonasnfkbpmmqya;bxxxiltdgdbjwmuddbxvpvkjaky
extendRoute;275;bxxxiltdgdbjwmuddbxvpvkjaky
addRoad;ovvtiycrjjhrfrpowpufamrwa;ajplfdfljikeafqlhclbkllxlrhvl;26;15
newRoute;187;bxxxiltdgdbjwmuddbxvpvkjaky;iajlzgkitf
newRoute;383;ovvtiycrjjhrfrpowpufamrwa;wcmcdsawqvsabzwm
removeRoad;bxxxiltdgdbjwmuddbxvpvkjaky;inwzyddfqonasnfkbpmmqya
addRoad;ajplfdfljikeafqlhclbkllxlrhvl;ajplfdfljikeafqlhclbkllxlrhvl;7;10
newRoute;756;bxxxiltdgdbjwmuddbxvpvkjaky;iajlzgkitf
removeRoad;inwzyddfqonasnfkbpmmqya;bxxxiltdgdbjwmuddbxvpvkjaky
extendRoute;512;iajlzgkitf
removeRoad;inwzyddfqonasnfkbpmmqya;bxxxiltdgdbjwmuddbxvpvkjaky
getRouteDescription;187
newRoute;486;yjlnkddrpeibszzsaudxyndqe;ajplfdfljikeafqlhclbkllxlrhvl
addRoad;dytyvtighulmna;ovvtiycrjjhrfrpowpufamrwa;22;12
repairRoad;wbkxadzwix;vlvcigpqvnstt;-5
repairRoad;ajplfdfljikeafqlhclbkllxlrhvl;inwzyddfqonasnfkbpmmqya;-2
addRoad;inwzyddfqonasnfkbpmmqya;inwzyddfqonasnfkbpmmqya;24;-4
removeRoad;yjlnkddrpeibszzsaudxyndqe;wcmcdsawqvsabzwm
extendRoute;158;dytyvtighulmna
addRoad;ovvtiycrjjhrfrpowpufamrwa;yjlnkddrpeibszzsaudxyndqe;8;-5
repairRoad;yjlnkddrpeibszzsaudxyndqe;bxxxiltdgdbjwmuddbxvpvkjaky;-2
getRouteDescription;486
extendRoute;187;ovvtiycrjjhrfrpowpufamrwa
extendRoute;275;vlvcigpqvnstt
getRouteDescription;383
getRouteDescription;383
removeRoad;ovvtiycrjjhrfrpowpufamrwa;dytyvtighulmna
getRouteDescription;187
newRoute;335;iajlzgkitf;dytyvtighulmna
removeRoad;wbkxadzwix;inwzyddfqonasnfkbpmmqya
repairRoad;vlvcigpqvnstt;dytyvtighulmna;-15
removeRoad;wcmcdsawqvsabzwm;yjlnkddrpeibszzsaudxyndqe
repairRoad;iajlzgkitf;ajplfdfljikeafqlhclbkllxlrhvl;-1
repairRoad;ajplfdfljikeafqlhclbkllxlrhvl;ovvtiycrjjhrfrpowpufamrwa;0
