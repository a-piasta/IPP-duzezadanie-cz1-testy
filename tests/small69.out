1: 
2: TAK
3: NIE
4: NIE
5: NIE
6: NIE
7: TAK
8: TAK
9: NIE
10: NIE
11: NIE
12: 
13: NIE
14: TAK
15: NIE
16: NIE
17: TAK
18: 
19: NIE
20: NIE
21: TAK
22: NIE
23: NIE
24: 
25: NIE
26: NIE
27: TAK
28: NIE
29: NIE
30: NIE
31: NIE
32: NIE
33: NIE
34: NIE
35: NIE
36: 
37: NIE
38: TAK
39: NIE
40: NIE
41: NIE
42: NIE
43: NIE
44: TAK
45: NIE
46: 
47: NIE
48: NIE
49: 
50: 
51: TAK
52: 
53: NIE
54: TAK
55: NIE
56: NIE
57: NIE
58: NIE
